docs/UserGuide.md

```diff
@@ -30,14 +30,14 @@ iVolunteer is your dedicated application for volunteer coordination, designed wi
    Some example commands you can try:
 
    * `elist` : Lists all events
-   
-   * `eadd n/food donation r/chef r/packer d/23-9-2023 1500 dsc/help food distribution m/50 potatoes b/50` : Creates an event with name `food donation`, roles needed `chef` and `packer`, event date `23rd September 2023, 3pm`, description `help food distribution`, materials needed `50 potatoes` and budget `$50`
+
+   * `ecreate n/food donation r/chef r/packer d/23/9/2023 1500 dsc/help food distribution m/50 potatoes b/50` : Creates an event with name `food donation`, roles needed `chef` and `packer`, event date `23rd September 2023, 3pm`, description `help food distribution`, materials needed `50 potatoes` and budget `$50`
```

Comment:
did you change the date format? preferably better to not create confusing formats since the slash is already there for the prefix

Reply:
i checked alr the `/` in the date doesn't affect the parsing and is more intuitive for the user compared to using `-`. but do u think we shld using `/` or `-`?